cmd/configschema/go.mod

```diff
@@ -1111,6 +1112,8 @@ replace github.com/openshift/api v3.9.0+incompatible => github.com/openshift/api
 // It appears that the v0.2.0 tag was modified.  Replacing with v0.2.1
 replace github.com/outcaste-io/ristretto v0.2.0 => github.com/outcaste-io/ristretto v0.2.1
 
+replace github.com/DataDog/datadog-agent/pkg/trace v0.47.0-rc.3 => github.com/DataDog/datadog-agent/pkg/trace v0.47.0-rc.3.0.20230720220350-143f7b52181b
```

Comment:
Why do we need this?

Reply:
`make update-otel` will unfortunately try to replace `v0.47.0-rc.3.0.20230720220350-143f7b52181b` with `v0.47.0-rc.3`. Unfortunately `v0.47.0-rc.3` doesn't contain the breaking change thus the downgrade performed by `update-otel` will always break datadog exporter. This blocked v0.82 so I had to revert the original PR.

Reply:
Discussed offline, I think we both feel like tagging a pre-release is a good option here